Added/updated Longmoor lashings and Lamentin Losers

TheLocalPub opened the pull request.

joewhite94 commented on worlds/TheLocalPub/LongmoorLashings/Prefabs/Characters/Factions/BLUFOR/US_Army/GreenBerets/Character_US_SF_Officer_P_LandNav.et:
```diff
@@ -0,0 +1,33 @@
+SCR_ChimeraCharacter : "{C0F45FA0C11853D1}Prefabs/Characters/Factions/BLUFOR/US_Army/GreenBerets/Character_US_SF_Officer.et" {
```
Are these landnav course prefabs used in the mission?

TheLocalPub commented:
Updated Lemantin losers to actually have QRF spawn, fixed mines not being armed, and turrets not being mountable.
I also added system messages that now work per side.

TheLocalPub commented:
Updated LongmoorLashings

Til-Weimann reviewed (approved):
Will fix a remaining issue later
